etc/helm/pachyderm/templates/console/deployment.yaml

```diff
@@ -160,10 +160,10 @@ spec:
         name: tmp
       - emptyDir: {}
         name: home
-      {{- if and .Values.pachd.tls.enabled .Values.global.customCaCerts }}
+      {{- if and .Values.proxy.tls.enabled .Values.global.customCaCerts }}
       - name: pachd-tls-cert
         secret:
-          secretName: {{ required "If pachd.tls.enabled, you must set pachd.tls.secretName" .Values.pachd.tls.secretName | quote }}
+          secretName: {{ required "If pachd.tls.enabled, you must set pachd.tls.secretName" .Values.proxy.tls.secretName | quote }}
```

Comment:
update the required comment to reference `proxy.tls.secretName`

Reply:
Good catch, thank you.